labs-ios-starter/Application/View/UI/StyleSheet.swift

```diff
@@ -26,9 +26,11 @@ extension UIColor {
         UIColor(named: named.rawValue)!
     }
 
-    static var background = getColor(.background)
     static var action = getColor(.action)
     static var shadow = getColor(.shadow)
+    static var accent = getColor(.accent)
```

Comment:
I think this is close to, but not overriding the actual accent color property. Since it's set on both variables, it will work either way